sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q13.sf100/explain.txt

```diff
@@ -110,16 +110,16 @@ Input [13]: [ss_sold_date_sk#1, ss_hdemo_sk#3, ss_addr_sk#4, ss_store_sk#5, ss_q
 (16) Scan parquet default.customer_address
 Output [3]: [ca_address_sk#18, ca_state#19, ca_country#20]
 Batched: true
-Location [not included in comparison]/{warehouse_dir}/date_dim]
-PushedFilters: [EqualTo(d_year,2001), IsNotNull(d_year), IsNotNull(d_date_sk)]
-ReadSchema: struct<d_date_sk:int,d_year:int>
+Location [not included in comparison]/{warehouse_dir}/customer_address]
+PushedFilters: [Or(Or(In(ca_state, [TX,OH]),In(ca_state, [OR,NM,KY])),In(ca_state, [VA,TX,MS])), IsNotNull(ca_country), EqualTo(ca_country,United States), IsNotNull(ca_address_sk)]
+ReadSchema: struct<ca_address_sk:int,ca_state:string,ca_country:string>
```

Comment:
@Ngone51 This is not the expected change, there is a bug in PlanStability?

Reply:
Is it still an issue? I see it's marked as `Outdated`.

Reply:
Thank you, it seems this is not an issue.